pkgs/os-specific/linux/opa-psm2/default.nix

```diff
@@ -0,0 +1,41 @@
+{ stdenv, fetchFromGitHub, numactl, pkgconfig }:
+
+stdenv.mkDerivation rec {
+  name = "opa-psm2-${version}";
+  version = "10_10_1_0_36";
```

Comment:
Other distributions call this package "libpsm2" (https://repology.org/project/libpsm2/information). 
Also the newest version according to github is 11.2.91 (note also dots instead of underscores).

Reply:
OK!
- now called libpsm2
- upgraded to the newest version, and using the psm tag instead of the ifs version
- created an ifs_version for the download